nightwatch.conf.js

```diff
@@ -65,19 +70,19 @@ module.exports = {
     'browserstack.local': {
       ...browserStack,
       desiredCapabilities: {
-       browserName: 'chrome',
-          'bstack:options' : {
-              "os" : "OS X",
-              "osVersion" : "Sierra",
-              "buildName" : "browserstack-build-1",
-              "sessionName" : "Selenium-4 Nodejs snippet test",
-              "source": "nightwatch:sample-master:v1.0",
```

Comment:
this got removed. also build name changed.

Reply:
ack